### ISSUE TITLE
Can't converge Loss function in object detection API

### ISSUE THREAD

**Issue by miladfa7** [tensorflow/models#8114]
I receive these classification loss spikes and **no convergence on my own data**.

![Screenshot from 2020-01-31 17-52-04](https://user-images.githubusercontent.com/25765644/73547148-9fab4f00-4453-11ea-8a45-2a115530129c.png)

the output `tensorboard --logdir training/`
<br>
![Screenshot from 2020-01-31 17-57-36](https://user-images.githubusercontent.com/25765644/73547005-670b7580-4453-11ea-98ab-3152c6e82321.png)

![Screenshot from 2020-01-31 17-57-46](https://user-images.githubusercontent.com/25765644/73547011-696dcf80-4453-11ea-9c28-01be614934bd.png)

What is the reason for this problem?

**Comment by asmaamirkhan** [comment]
- 🤔 It is a normal result since step number is still 500 
- 👀 You have to wait for at least 2000 steps (if your data is complex it will take more)
